src/config/SourcegraphConfigOptions.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+final class PhabricatorSourcegraphConfigOptions extends PhabricatorApplicationConfigOptions
+{
+    public function getName()
+    {
+        return pht('Sourcegraph');
+    }
+
+    public function getDescription()
+    {
+        return pht('Configure Sourcegraph Settings.');
+    }
+
+    public function getIcon()
+    {
+        return 'fa-asterisk';
+    }
+
+    public function getGroup()
+    {
+        return 'apps';
+    }
+
+    public function getOptions()
+    {
+        $repo_type = 'sourcegraph.repo';
+        $repo_example = array(
+            array(
+                'path' => 'gorilla/mux',
```

Comment:
The repo path `gorilla/mux` is not actually standard (ie it's not what most Sourcegraph customers would have for that repo); the standard value would be `github.com/gorilla/mux`. If the goal is to demonstrate a non-standard repo path, then you should make that clear somehow. Otherwise use `github.com/gorilla/mux` (which I think makes more sense, and site admins with non-standard paths will still grok it).